Header header

string[] name
float64[] jointAccByDerivative
float64[] filteredJointAccByDerivative
float64[] jointAccByIMU
float64[] filteredJointAccByImu
float64[] filteredBias
float64[] mergedJointAcc
